src/commands/ms/add.ts

```diff
@@ -1,8 +1,8 @@
 import { CliUx, Command, Flags } from '@oclif/core'
-import { MicroService } from '../../models/bundle-descriptor'
-import { MicroServiceStack } from '../../models/component'
+import { Microservice } from '../../models/bundle-descriptor'
```

Comment:
@raxkaynan I think we agreed to leave `MicroService` in the code. Is there a reason for this?

Reply:
Ok we agreed on renaming as found on Wikipedia and other official sources